python/cuml/cuml/svm/svc.pyx

```diff
@@ -581,19 +582,7 @@ class SVC(SVMBase,
             self.dtype,
         )
 
-        # If sample_weight is not None, it is a cupy array, and we need to
-        # convert it to a numpy array for sklearn
         if sample_weight is not None:
```

Comment:
That should be always true now, no?

Reply:
I missed the early return on [line 159](https://github.com/jcrist/cuml/blob/29ce6c7239d43dc2ec45476b1f0b68d6f8b0d549/python/cuml/cuml/svm/svc.pyx#L159).